homeassistant/components/dunehd/media_player.py

```diff
@@ -26,6 +28,10 @@
     | MediaPlayerEntityFeature.PREVIOUS_TRACK
     | MediaPlayerEntityFeature.NEXT_TRACK
     | MediaPlayerEntityFeature.PLAY
+    | MediaPlayerEntityFeature.PLAY_MEDIA
+    | MediaPlayerEntityFeature.VOLUME_MUTE
+    | MediaPlayerEntityFeature.VOLUME_STEP
```

Comment:
Let's set this in the PR which fixes volume since volume control is broken currently.

Reply:
That wasn't meant to be there indeed. Fixed now.